pages/common/zipnote.md

```diff
@@ -0,0 +1,17 @@
+# zipnote
+
+> View, add, or edit a zip archive's comments.
+> Files can also be renamed in the zip archive.
+> More information: <https://manned.org/zipnote>.
+
+- View the comments on a zip archive:
+
+`zipnote {{path/to/file.zip}}`
+
+- Extract the comments on a zip archive to a file:
+
+`zipnote {{path/to/file.zip}} > {{path/to/file.txt}}`
+
+- Add/Update comments/filename on/in a zip archive from a file:
```

Comment:
why say `filename` here?

Reply:
I will correct it. It should not be adding or updating a filename.